source/quick-start.txt

```diff
@@ -13,112 +20,146 @@ Quick Start
 
-This guide shows you how to add the MongoDB Kotlin driver dependencies using
-Gradle or Maven. We recommend that you use an integrated development
-environment (IDE) such as Intellij IDEA or Eclipse IDE to make it more convenient
-to configure Gradle or Maven to build and run your project.
+This guide shows you how to add the MongoDB Kotlin driver dependencies
+by using Gradle or Maven. We recommend that you use an integrated development
+environment (IDE) such as IntelliJ IDEA or Eclipse IDE to configure
+Gradle or Maven to build and run your project.
 
-If you are not using an IDE, see
-`Creating New Gradle Builds <https://guides.gradle.org/creating-new-gradle-builds/>`_
-or  `Building Maven <https://maven.apache.org/guides/development/guide-building-maven.html>`_
+If you are not using an IDE, see the
+`Creating New Gradle Builds
+<https://guides.gradle.org/creating-new-gradle-builds/>`__ guide 
+or the `Building Maven
+<https://maven.apache.org/guides/development/guide-building-maven.html>`__ guide
 for more information on how to set up your project.
 
 .. _add-mongodb-dependency:
 
 Add MongoDB as a Dependency
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you are using `Gradle <https://gradle.org/>`__, add the following to your
-``build.gradle.kts`` dependencies list:
+If you are using `Gradle <https://gradle.org/>`__ to manage your
+packages, add the following entry to your ``build.gradle.kts``
+dependencies list:
 
 .. include:: /includes/kotlin-driver-coroutine-gradle-versioned.rst
 
-If you are using `Maven <https://maven.apache.org/>`__, add the following to
-your ``pom.xml`` dependencies list:
+If you are using `Maven <https://maven.apache.org/>`__ to manage your
+packages, add the following entry to your ``pom.xml`` dependencies list:
 
 .. include:: /includes/kotlin-driver-coroutine-maven-versioned.rst
 
-Once you configure your dependencies, ensure they are available to your
-project which may require running your dependency manager and refreshing
-the project in your IDE.
+Once you configure your dependencies, ensure that they are available to your
+project by running the dependency manager and refreshing the
+project in your IDE.
+
+Add Serialization Library Dependencies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable the driver to convert between Kotlin objects and BSON, the
+data format for documents in MongoDB, you must also add one or both of the
+following serialization packages to your application:
+
+- ``bson-kotlinx`` *(Recommended)*
+- ``bson-kotlin``
+
+If you are using Gradle to manage your packages, add one of the following
+entries to your ``build.gradle.kts`` dependencies list: 
+
+.. include:: /includes/serialization-libs-gradle-versioned.rst
+
+If you are using Maven to manage your packages, add one of the following
+entries to your ``pom.xml`` dependencies list:
+
+.. include:: /includes/serialization-libs-maven-versioned.rst
+
+Once you configure your dependencies, ensure that they are available to your
```

Comment:
[Style Guide for once/after](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-after)

```suggestion
After you configure your dependencies, ensure that they are available to your
```

Reply:
changed all instances